### PR TITLE
Clean incoming deviceTokens with a regexp

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Description

 none

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204108581976371

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- removed: Unused legacy databases. The v1 routes continue to support security message registration, but won't remember any preferences.
```
Comment (peachbits) [inline CHANGELOG.md:5]: This line appears to be for a different PR

**Review by peachbits — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (swansontec) [reply CHANGELOG.md:5]: It is, but it somehow got lost.
